estela-web/README.md

```diff
@@ -1,6 +1,36 @@
-<h1 align="center">estela Web</h1>
+<h1 align="center">
+    <span>
+        <img width="38" style="vertical-align: middle" src="https://cdn-icons-png.flaticon.com/512/1078/1078191.png">
+    </span>
+    estela Web
+</h1>
 
-This project is created using React Framework (along with Ant Design) and Typescript. This module implements a
-user-friendly environment that communicates with the estela API and lets you manage your spiders and scraping
-projects. For more information on this module and how to set it up, please refer to our
-[official documentation](https://bitmaker.la/docs/bitmaker-cloud/web.html).
+![version](https://img.shields.io/badge/version-0.1-blue)
+![yarn-version](https://img.shields.io/badge/yarn-v1.22.19-blue)
+![build](https://img.shields.io/badge/build-passing-brightgreen)
+![CI](https://github.com/eslint/eslint/workflows/CI/badge.svg)
+
+<p align="center" style="margin-top: 20px">
+  <a href="https://reactjs.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/react/react-original-wordmark.svg" alt="react" width="40" height="40"/> </a>
+  <a href="https://nodejs.org/" target="_blank" rel="noreferrer"> <img src="https://www.vectorlogo.zone/logos/nodejs/nodejs-icon.svg" alt="nodejs" width="40" height="40"/> </a>
+</p>
+
+#### The project is created using [React Framework](https://reactjs.org) and [Typescript](https://www.typescriptlang.org). 
+
+This module introduces users with a friendly interface that interacts with the [estela API 🕷](https://github.com/bitmakerla/estela/tree/main/estela-api)  and lets you manage your spiders and scraping projects.
+
+The module allows users to **CREATE**, **UPDATE** and **DELETE** information and data collected from your spiders. Similar to [estela-cli](https://github.com/bitmakerla/estela-cli) but, with an interactive interface.
```

Comment:
I think we can be more direct by saying "This module allows users to **manage their scraping projects** and data collected from their spiders. This is a **graphical alternative** to `estela-cli`."